{"name": "test_login", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TimeoutError: Page.goto: Timeout 30000ms exceeded.\nCall log:\n  - navigating to \"https://app.clickup.com/login\", waiting until \"load\"", "trace": "tests\\ui\\conftest.py:27: in logged_in_page\n    login_page.login('milanakan2001@gmail.com', 'Milakan2307')\npages\\login_page.py:19: in login\n    self.navigate_to()\npages\\base_page.py:16: in navigate_to\n    self.page.goto(full_url)\n.venv\\Lib\\site-packages\\playwright\\sync_api\\_generated.py:9020: in goto\n    self._sync(\n.venv\\Lib\\site-packages\\playwright\\_impl\\_page.py:552: in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n.venv\\Lib\\site-packages\\playwright\\_impl\\_frame.py:145: in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py:61: in send\n    return await self._connection.wrap_api_call(\n.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py:528: in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE   playwright._impl._errors.TimeoutError: Page.goto: Timeout 30000ms exceeded.\nE   Call log:\nE     - navigating to \"https://app.clickup.com/login\", waiting until \"load\""}, "description": "Проверка успешной авторизации и наличия элемента 'Board' после входа.", "start": 1744745715867, "stop": 1744745715867, "uuid": "a77e81cd-b001-4590-b92f-70de52cf79d1", "historyId": "2576a7dec22103167e50834d833c3fcb", "testCaseId": "2576a7dec22103167e50834d833c3fcb", "fullName": "tests.ui.test_login_page#test_login", "labels": [{"name": "feature", "value": "Авторизация"}, {"name": "parentSuite", "value": "tests.ui"}, {"name": "suite", "value": "test_login_page"}, {"name": "host", "value": "DESKTOP-G6JEM9V"}, {"name": "thread", "value": "10216-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.ui.test_login_page"}]}